core/domain/caching_services.py

```diff
@@ -212,13 +221,15 @@ def set_multi(namespace, sub_namespace, id_value_mapping):
 
     memory_cache_id_value_mapping = {
         _get_memcache_key(namespace, sub_namespace, obj_id):
-        SERIALIZATION_FUNCTIONS[namespace](value)
+        SERIALIZATION_FUNCTIONS[namespace](value) # type: ignore[operator]
```

Comment:
Why is this ignore needed?

Reply:
this also maps functions as a key-value pair. so, passing value as a argumment gives error `Cannot call function of unknown type  [operator] `

Reply:
Done!